<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="open_image_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_path">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>941</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Pick a file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>34</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_gpu_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Bands</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>303</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="select_B1">
       <property name="text">
        <string>B1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B2">
       <property name="text">
        <string>B2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B3">
       <property name="text">
        <string>B3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B4">
       <property name="text">
        <string>B4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B5">
       <property name="text">
        <string>B5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B6">
       <property name="text">
        <string>B6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B7">
       <property name="text">
        <string>B7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>B8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B9">
       <property name="text">
        <string>B9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B10">
       <property name="text">
        <string>B10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="select_B11">
       <property name="text">
        <string>B11</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="k_means_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>K-Means GPU</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>1181</width>
      <height>871</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="image_area">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>8011</width>
       <height>8111</height>
      </rect>
     </property>
     <widget class="QLabel" name="preview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>8011</width>
        <height>8111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Classes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Itterations:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="k_means_button_cpu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>K-Means CPU</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="classes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="itterations">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>open_image_button</zorder>
   <zorder>file_path</zorder>
   <zorder>label</zorder>
   <zorder>load_gpu_button</zorder>
   <zorder>k_means_button</zorder>
   <zorder>scrollArea</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>k_means_button_cpu</zorder>
   <zorder>classes</zorder>
   <zorder>itterations</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
